x/subscription/keeper/subscription.go

```diff
@@ -169,32 +164,31 @@ func (k Keeper) CreateSubscription(
 		// new subscription needs a default project
 		err = k.projectsKeeper.CreateAdminProject(ctx, consumer, plan)
 		if err != nil {
-			details := map[string]string{
-				"err": err.Error(),
-			}
-			return utils.LavaError(ctx, logger, "CreateSubscription", details, "failed to create default project")
+			return utils.LavaFormatError("failed to create default project", err)
```

Comment:
should be a warning, inner functions should decide on warning or error for different scenarious

Reply:
done